scrapers/HelixStudios.yml

```diff
@@ -0,0 +1,82 @@
+name: "HelixStudios"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/video/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title: //title/text()
+      Date:
+        selector: //div[@class="info-items"]/span[@class="info-item date"]/text()
+        postProcess:
+          - replace:
+              - regex: th
+                with:
+              - regex: st
+                with:
+              - regex: rd
+                with:
+          - parseDate: Jan 02, 2006
+      Details:
+        selector: //div[@class="description-content less"]/p/text()
+      Tags:
+        Name: //div[@class="video-tags-wrapper"]/a/text()
+      Performers:
+        Name:
+                selector: //div[@class="thumbnail model-thumbnail circle-thumbnail"]/div/div/img/@alt
+      Image:
+        selector: //video/@poster
+      Studio:
+        Name:
+          fixed: Helix
+
+  performerScraper:
+    performer:
+      Name: //div[contains(@class, "top")]/h1/text()
+      Gender:
+        fixed: Male
+      Height:
+        selector: //span[text()="Height"]/following-sibling::text()
+        postProcess:
+          - feetToCm: true
+      Measurements:
+        selector: //span[text()="Cock"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "0\'"
+          - replace:
+              - regex: inch*
+                with: "\'\'"
+          - feetToCm: true
```

Comment:
should be
```yaml
        postProcess:
          - replace:
              - regex: ^
                with: "0ft"
          - replace:
              - regex: inch.*
                with: 
          - feetToCm: true
  ```
Note: due to feetToCm limitations the fractional part of the measurement is ignored eg 6.5 inches -> 6 inches